### DISCUSSION TITLE
国际服生息演算报错

### DISCUSSION THREAD

**Discussion by Toddie-Lubert** [MaaAssistantArknights/MaaAssistantArknights#11447] [Q&A] [answered]
请问国际服生息演算点数没刷满，只刷一轮就停在图1页面，手动点击进入第4天后就报错显示任务已完成（如图2），如何解决？
<img width="1287" alt="issue" src="https://github.com/user-attachments/assets/2d1d9c58-de5c-445f-b4d1-c7b16561a24d" />
<img width="1287" alt="issue2" src="https://github.com/user-attachments/assets/72933c85-c6e7-4645-a88d-b5e812ecf014" />

**Comment by ABA2396** [comment, marked as answer]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new/choose
